test/integration_test.js

```diff
@@ -648,6 +648,80 @@ describe("Integration tests", function() {
             });
           });
         });
+
+        describe("Resolving conflicts doesn't interfere with sync", () => {
+          const conflictingId = uuid4();
+          const testData = {
+            localSynced: [
+              {id: conflictingId, title: "conflicting task", done: false}
+            ],
+            localUnsynced: [],
+            server: [],
+          };
+          beforeEach(() => testSync(testData));
+          it("should sync over resolved records", () => {
+            return tasks.update({id: conflictingId, title: "locally changed title"},
+                         {patch: true})
+              .then(newRecord => {
+                expect(newRecord.data.last_modified).to.exist;
+                return tasks.api.bucket("default").collection("tasks").updateRecord(
+                  {id: conflictingId, title: "remotely changed title"}, {patch: true});
+              })
+              .then(() => tasks.sync())
+              .then(syncResult => {
+                expect(syncResult.ok).eql(false);
+                expect(syncResult.conflicts).to.have.length.of(1);
+                // always pick our version, which has an older last_modified
```

Comment:
> which has an older last_modified

Instead of this, I would note that `resolve()` is going to take the remote last_modified